developmentnow/isATrainBlockingEleventh
# As/click firestore updates

**What Changed:** 

- Moved getTrain call to componentDidMount from render
- Addded method to update DB in fire class
- Added additional logging
- removed required propType in layout which caused an error
- changed naming of _train method to _getTrain

**How To Test**
- Pull down branch
- Open debugger, click buttons, watch as DB values toggle from true to false depending on the button clicked.  Check this link (https://console.firebase.google.com/u/0/project/istrainblocking11th/database/firestore/data~2Ftrain~2Fblocked)
